pom.xml

```diff
@@ -60,9 +60,8 @@
 		<releaseExt>-preview</releaseExt>
 
 		<!-- Driver Dependencies -->
-		<azure.keyvault.version>1.2.4</azure.keyvault.version>
-		<azure.adal4j.version>1.6.5</azure.adal4j.version>
-		<rest.client.version>1.7.4</rest.client.version>
+		<azure.keyvault.version>4.1.4</azure.keyvault.version>
```

Comment:
Could you confirm that these are production release versions?

Reply:
Yes, these are production release versions but since we had made these changes a while ago, the versions were from a previous release. We now have new releases and I will update the PR to use the latest versions.